.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "scikit-image devcontainer",
+	"build": {
+		"context": "..",
+		"dockerfile": "Dockerfile"
+	},
+	"features": {
+		"ghcr.io/devcontainers-contrib/features/pre-commit:2": {}
+	},
+	"postCreateCommand": "pre-commit install && bash ./.devcontainer/postBuild",
```

Comment:
I was surprised that pre-commit worked before the development setup in `postBuild`. Seems like it is included in the container by default
```sh
$ which pre-commit
/usr/local/py-utils/bin/pre-commit
```